src/Data/Cycles.ts

```diff
@@ -267,7 +268,7 @@ function updateNodeList(cycle: P2PTypes.CycleCreatorTypes.CycleData): void {
 
   NodeList.addNodes(NodeList.NodeStatus.SYNCING, consensorInfos)
 
-  NodeList.setStatus(NodeList.NodeStatus.ACTIVE, activatedPublicKeys)
+  NodeList.setStatus(NodeList.NodeStatus.ACTIVE, activatedPublicKeys);
```

Comment:
nitpick: prettier removed `;` if I am correct
```suggestion
  NodeList.setStatus(NodeList.NodeStatus.ACTIVE, activatedPublicKeys)
```